test/test-deployment-redirects.js

```diff
@@ -56,11 +71,15 @@ const testRow = (row) => {
 
       const matched = redirect === row.target
       if (matched) {
-        console.log(chalk.green('  Matched'))
+        console.log(chalk.green(`  Matched: ${redirect}`))
+
       } else {
-        console.error(chalk.red('  Failed to match:'))
-        console.error(chalk.red(`    Expected redirect: ${row.target}`))
-        console.error(chalk.red(`    Actual redirect:   ${redirect}`))
+        let partialMatch = redirect.indexOf(row.target) === 0 ? 'starts-with' : false
```

Comment:
what case would this match on?

Reply:
Added inline comment. Comes up a lot where the expected URL contains an extra query param that is not present on the actual redirect.